app/src/main/java/com/lee989898/soptlee/signup/SignUpActivity.kt

```diff
@@ -0,0 +1,49 @@
+package com.lee989898.soptlee.signup
+
+import android.app.Activity
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import androidx.databinding.DataBindingUtil
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import com.lee989898.soptlee.R
+import com.lee989898.soptlee.signin.SignInActivity
+import com.lee989898.soptlee.databinding.ActivitySignUpBinding
+
+class SignUpActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivitySignUpBinding
+    lateinit var signUpViewModel: SignUpViewModel
```

Comment:
```suggestion
    private val viewModel by viewModels<SignUpViewModel>()
```
Android KTX를 사용하면 뷰모델 위임 초기화로 좀 더 깔끔하게 생성할 수 있습니다.

Reply:
꿀팁 감사합니다